### PR TITLE
Fix docker compose failing start issue

### PR DESCRIPTION
**Pull request by Maxpeng92**
#179

### CODE REVIEW

File: docker/docker-compose.yml [hunk 1]
```diff
@@ -12,7 +12,9 @@ services:
   nginx:
     image: nginx:1.13-alpine
     container_name: mainflux-nginx
-    network_mode: bridge
+    #network_mode: bridge
```
Comment (drasko) [inline docker/docker-compose.yml]: Drop comments (dead code)

File: docker/docker-compose.yml [hunk 2]
```diff
@@ -26,12 +28,16 @@ services:
   nats:
     image: nats:1.0.2
     container_name: mainflux-nats
-    network_mode: bridge
+    #network_mode: bridge
```
Comment (drasko) [inline docker/docker-compose.yml]: Same here

File: docker/docker-compose.yml [hunk 3]
```diff
@@ -26,12 +28,16 @@ services:
...
+    restart:
+      on-failure
 
   postgres:
     image: postgres:10.2-alpine
     container_name: mainflux-postgres
-    network_mode: bridge
+    #network_mode: bridge
```
Comment (drasko) [inline docker/docker-compose.yml]: Same here

**Review by drasko — CHANGES_REQUESTED** [review]
Please drop the unused comments

**Review by drasko — APPROVED** [review]
LGTM

**Comment by mijicd** [comment]
Merged, since it is a slight improvement in compose, though it does not solve the problem.

**Pull request merged**